contents/docs/features/organizations.md

```diff
@@ -11,6 +11,13 @@ An organization is the largest building block of PostHog's structure: it contain
 
 By default an organization only has the default project. For multiple projects – which are a premium PostHog feature – [take a look at their dedicated page](/docs/features/multiple-projects).
 
+### Segregating data into multiple projects
```

Comment:
Title Case?

Reply:
Yeah I'm a fan of title case and always push for it, but I've started to stop doing that for long subtitles